package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@eppo/js-client-sdk-common",
-  "version": "3.3.1",
+  "version": "3.3.2",
```

Comment:
_technically_, this is a breaking change since we're removing an exported type (IEppoClient), but it should only be an issue for our internal libraries that have `@eppo/js-client-sdk-common` as a dependency, and those libraries use the exact version (rather than a version range) of this dependency in their package.json. So with that all said, I kinda feel like a major version bump is unnecessary (assuming we're following semver)

Reply:
Actually, I'll just wait to merge this in until the "Evaluation Reasons" project also gets merged in